lib/nesta/commands.rb

```diff
@@ -158,28 +158,33 @@ def initialize(*args)
           name.nil? && (raise UsageError.new('name not specified'))
           @name = name
           @gem_name = "nesta-plugin-#{name}"
+          @namespace_path = namespace_path(@gem_name)
           if File.exist?(@gem_name)
             raise RuntimeError.new("#{@gem_name} already exists")
           end
         end
 
+        def namespace_path(name)
+          name.tr('-', '/')
+        end
+
         def lib_path(*parts)
           File.join(@gem_name, 'lib', *parts)
         end
 
         def modify_required_file
-          File.open(lib_path("#{@gem_name}.rb"), 'w') do |file|
+          File.open(lib_path("#{@namespace_path}.rb"), 'w') do |file|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.